src/content/post/20161129_git_reflog_amend.md

```diff
@@ -0,0 +1,114 @@
++++
+date        = "2016-11-29T08:51:09+02:00"
+title       = "Trapped by an amend, saved by a reflog"
+tags        = ['git']
+topics      = ['git']
+description = "Personal experience on getting in trouble with a common git command and finding my way out."
+slug        = 'trapped-by-amend-saved-by-reflog'
+author      = "Marcello"
++++
+[SparkFabrik]: http://www.sparkfabrik.com/  "SparkFabrik"
+
+We use Git on a daily basis, and most of the time we always enter the same commands.
+It can happen to found ourselves trapped by one of them, sometimes because we forgot a side-effect, other times because we confused the right option.
+That's OK, because within Git most of the time lies the appropriate and easiest solution. So, let me tell you how I was hit by an `--amend` during a commit and how I easily got out.
+
+<!--more-->
+
+Sometimes during the day, especially in the time right before lunch or going home, it's possible to forget one the most important rules of the savvy developer: in the terminal, you can be quick with all the keys, but think twice before pressing "enter".
+
+Breaking this rule is what happened to me some days ago, but that was luckily a chance to remind me how much powerful git is and how sometimes the solutions to an apparently bad situation can be just a few commands away.
```

Comment:
Adverbs go first: `but that was luckily a chance` -> `but luckily that was a chance`.

If you mind I'd also avoid the subordinate: `..., but...` -> `. <new sentence>`. This is just a matter of style though. Keep your version if it "sounds like you".